client/network/src/protocol/message.rs

```diff
@@ -232,8 +232,8 @@ pub mod generic {
 		pub receipt: Option<Vec<u8>>,
 		/// Block message queue if requested.
 		pub message_queue: Option<Vec<u8>>,
-		/// Justification if requested.
-		pub justification: Option<Justification>,
+		/// Justifications if requested.
+		pub justifications: Option<Justifications>,
```

Comment:
That's a silent breaking change, isn't it?
This code isn't even actively in use anymore and is kept purely for backwards compatibility, so changing it kills the entire point of it.

Reply:
Good catch, thanks! Reverted this change